pxd.c

```diff
@@ -874,7 +874,8 @@ struct pxd_device* find_pxd_device(struct pxd_context *ctx, uint64_t dev_id)
 	return pxd_dev;
 }
 
-static int __pxd_update_path(struct pxd_device *pxd_dev, struct pxd_update_path_out *update_path);
+static int __pxd_update_path(struct pxd_device *pxd_dev, bool can_failover,
```

Comment:
can_failover can be embedded in pxd_update_path_out just like pxd_add_ext_out.

Reply:
yes can be embedded and is more appropriate. revised.